python3/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM socrata/base
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install python3 python3-dev python3-pip
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.4 1
```

Comment:
I can't remember how to make "python" refer to python3 instead of python2. Is this what you're trying to get at here?

I do know that python2 and python3 are not alternatives...